[iOS] build warning (14.0 alpha)

```
[06:11:50][Step 1/3] /Users/mcdurdin/buildAgent/work/11799982e0502208/keyman/ios/engine/KMEI/KeymanEngine/Classes/Resource Management/ResourceDownloadManager.swift:330:90: warning: cast from 'Error' to unrelated type 'String' always fails
[06:11:50][Step 1/3]         log.info("Failed to fetch lexical model list for "+languageID+". error: "+(error as! String))
[06:11:50][Step 1/3]                                                                                    ~~~~~ ^   ~~~~~~
```

https://build.palaso.org/viewLog.html?buildId=200944&buildTypeId=Keyman_iOS_Master&tab=buildLog&_focus=339#_state=70

--- a/ios/engine/KMEI/KeymanEngine/Classes/Resource Management/ResourceDownloadManager.swift
+++ b/ios/engine/KMEI/KeymanEngine/Classes/Resource Management/ResourceDownloadManager.swift
@@ -327,7 +327,7 @@ public class ResourceDownloadManager {
     //get list of lexical models for this languageID  /?q=bcp47:en
     func listCompletionHandler(lexicalModels: [LexicalModel]?, error: Error?) -> Void {
       if let error = error {
-        log.info("Failed to fetch lexical model list for "+languageID+". error: "+(error as! String))
+        log.info("Failed to fetch lexical model list for "+languageID+". error: "+error.localizedDescription)
         downloader.downloadFailed(forLanguageID: languageID, error: error)
       } else if nil == lexicalModels {
         //TODO: put up an alert instead

--- a/web/source/osk/visualKeyboard.ts
+++ b/web/source/osk/visualKeyboard.ts
@@ -657,6 +657,11 @@ namespace com.keyman.osk {
      * @return      {Object}                          fully formatted OSK object
      */
     deviceDependentLayout(keyboard: keyboards.Keyboard, formFactor: utils.FormFactor): HTMLDivElement {
+      if(!keyboard) {
+        // May occasionally be null in embedded contexts; have seen this when iOS engine sets
+        // keyboard height during change of keyboards.
+        keyboard = new keyboards.Keyboard(null);
+      }
       let layout = keyboard.layout(formFactor);
       let util = com.keyman.singleton.util;
       let oskManager = com.keyman.singleton.osk;

